src/main/java/seedu/address/logic/commands/LogCommand.java

```diff
@@ -0,0 +1,35 @@
+package seedu.address.logic.commands;
+
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+
+/**
+ * Saves the results of a FindCommand to the logbook.
+ */
+public class LogCommand extends Command {
+
+    public static final String COMMAND_WORD = "log";
+
+    public static final String COMMAND_WORD_ALIAS = "lo";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + "or" + COMMAND_WORD_ALIAS
+            + ": Saves the results of the most recent FindCommand to the logger tab.\n"
+            + "Example 1: " + COMMAND_WORD + "\n"
+            + "Example 2: " + COMMAND_WORD_ALIAS
+
+    public static final String MESSAGE_SUCCESS = "Results of the FindCommand have been saved to the logger tab.";
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        // Ensure that there are results from the most recent FindCommand
+        if (model.getFoundPersonsList().isEmpty()) {
+            throw new CommandException(Messages.MESSAGE_EMPTY_FIND_RESULT);
+        }
+
+        model.getLogBook().setPersons(model.getFoundPersonsList());
```

Comment:
In retrospect, we may have been too hasty in approving the `LogBook` implementation. This is likely setting the `LogBook` internal list to follow the `ObservableList` instance of `AddressBook`.